Makefile

```diff
@@ -11,12 +11,13 @@ GCC = g++
 endif
 LD = $(GCC)
 
+DEBUG_LD=lld
```

Comment:
Now it's possible to change default ld by passing DEBUG_LD=... (ifndef / endif is not really required).